Gemfile

```diff
@@ -2,8 +2,10 @@ source 'https://rubygems.org'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.6'
-# Use MySQL as the database for Active Record
-gem 'mysql2'
+# Use Postgres as the database for Active Record
```

Comment:
Sorry I'm not up to date on all the changes. What was the reason for changing the Active Record database to Postgres?  Was it because it worked better for deployment with Unicorn and Heroku?